test/benchmark/bench_linux_and_aster.sh

```diff
@@ -88,8 +88,8 @@ run_benchmark() {
 
     echo "Parsing results..."
     local linux_avg aster_avg
-    linux_avg=$(awk "/${search_pattern}/{print \$$result_index}" "${linux_output}" | tr -d '\r')
-    aster_avg=$(awk "/${search_pattern}/{print \$$result_index}" "${aster_output}" | tr -d '\r')
+    linux_avg=$(awk "/${search_pattern}/{print \$$result_index}" "${linux_output}" | tr -d '\r' | tail -n 1)
+    aster_avg=$(awk "/${search_pattern}/{print \$$result_index}" "${aster_output}" | tr -d '\r' | tail -n 1)
```

Comment:
Narrowing the scope of the result to the last line of output conflicts with the use of awk pattern, whose purpose is to find the line where the result can be found. And it also conflicts with the current documentation.

Reply:
You are right. Now I replace it with `END` method, which has no influence on other benchmarks.